sdc/crypto/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '0.2.3'
+__version__ = '0.2.4'
```

Comment:
This should be a minor version change. It's a fairly significant change (and is breaking).

Reply:
Done